src/p4p/test/test_nt.py

```diff
@@ -267,18 +267,41 @@ def testSubStore(self):
         self.assertEqual(V.a.value.index, 1)
         self.assertEqual(V.b.value.index, 0)
 
-    def testWrap(self):
+    def testDictionaryOfValueWrap(self):
         W = nt.NTEnum()
         V = W.wrap({'index':1, 'choices':['X','Y']})
```

Comment:
This existing test fails because it goes through the `isinstance(value, dict)` branch in NTEnum.wrap(), which now expects 'value.index' and/or 'value.choices' like NTScalar.wrap() does. Should I update this test, or should NTEnum.wrap() be able to handle a dictionary that only contains fields meant for the Value.value?